src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineInstance.java

```diff
@@ -97,7 +98,7 @@ protected void _terminate(TaskListener listener) throws IOException, Interrupted
           && this.createSnapshot
           && computer != null
           && !computer.getBuilds().failureOnly().isEmpty()) {
-        LOGGER.log(Level.INFO, "Creating snapshot for node ... " + this.getNodeName());
+        LOGGER.log(Level.INFO, CREATING_SNAPSHOT_FOR_NODE + this.getNodeName());
```

Comment:
I'm not sure making this a constant adds much value, as it's only referenced here.

Reply:
It's also used in the Snapshot tests, but in line with the feedback you gave about Strings being self documenting I think I could change this back to a literal and use literals in the tests. The only drawback being that changes here would break the SnapshotCreatedIT and would silently break for NoSnapshotCreatedIT since that tests for the absence of a log message.

Reply:
We should not test by checking logs in first place so problem seems not big..
As I mentioned for me all tests that are checking something in logs could be safely deleted/replaced by checking state..